setup.py

```diff
@@ -284,7 +284,7 @@ def run(self):
     # setup.py develop should check for submodules
     from setuptools.command.develop import develop
     setup_args['cmdclass']['develop'] = require_submodules(develop)
-    setup_args['cmdclass']['bdist_wheel'] = css_js_prerelease(get_bdist_wheel())
+    setup_args['cmdclass']['bdist_wheel'] = css_js_prerelease(get_bdist_wheel(), strict=False)
```

Comment:
Is there anywhere we're not setting `strict=False`? Does it make sense to make that the default, or even remove the parameter entirely?